web/index.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html lang="en-us">
+<head>
+    <meta charset="UTF-8">
+    <link href="css/styles.css" rel="stylesheet" type="text/css">  
+    <link href="css/buttons.css" rel="stylesheet" type="text/css">
+    <title>Simple Website</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    
+</head>
+<body>
+    <nav class="navbar navbar-expand navbar-light bg-light flex-column flex-md-row pipe-navbar justify-md-content-between">
+        <a class="navbar-brand" href="/"><img src="images/nus-logo.svg" width="66" height="30" class="d-inline-block" /> COVID-19 Question Answering engine</a>
```

Comment:
can we just add the link to nus logo instead of uploading the svg file?

Reply:
sure, done.